### PR TITLE
rackup error

### PR DESCRIPTION
**Pull request by mel-d**

### CODE REVIEW

**Comment by mel-d** [comment]
ActiveRecord::NoDatabaseError at /
FATAL: database "postgres://localhost/debugging-challenge" does not exist

```
file: postgresql_adapter.rb location: rescue in connect line: 661 
```

**Comment by sumeetjain** [comment]
Did you run `bin/setup`?

**Comment by mel-d** [comment]
Yes

**Comment by mel-d** [comment]
Ok, I'm attempting to get through the errors I'm shown while clicking directly on the assignment links from the welcome page--I get the 'Sinatra doesn't know this ditty'. I think this has either something to do with an error on the assignments part of admin_controller, or from the link that's on the welcome page. My most specific question is that I don't really understand .id, and I'm wondering whether the assignment.id part of the link might be causing my issues.

**Comment by mel-d** [comment]
(or, maybe, if Ruby maybe can't be used at all within links?)

**Comment by sumeetjain** [comment]
Think about what you know of routes and controller action so far. When I click a link to `"/about"`, how do I know that will happen next?

**Comment by mel-d** [comment]
It'll look for something named /about in the controller page--so this issue could be stemming from my :num code?

**Comment by sumeetjain** [comment]
> It'll look for something named /about in the controller page

Right. It's looking for a controller action with a path of "/about". Or something that matches that.

So what matches that for the link that's being clicked in our application?

**Comment by mel-d** [comment]
I'll check into that next--thanks!

**Comment by mel-d** [comment]
This runs without any bugs for me--I've submitted it for assignment approval, but I haven't added any extra features yet because I'd like to go keep chugging along on the string segmenter page first (if that's alright with you).

**Comment by sumeetjain** [comment]
NICE!

Yes, your plan is great–go for it.

**Comment by sumeetjain** [comment]
Wait, I think you missed some things.

File: app/views/admin/student.erb [hunk 1]
```diff
@@ -16,7 +16,7 @@ Back
     <% @assignments.each do |a| %>
       <tr>
         <td><a href="/assignments/<%= a.id %>"><%= a.title %></a></td>
-        <td><%= turned_in_assignment? %></td>
+        <td><%= @student.turned_in_assignment?(a) %></td>
 
         <!-- The score and adjusted scores aren't here at all. -->
         <!-- It's for you to complete on your own. -->
```
Comment (sumeetjain) [inline app/views/admin/student.erb:22]: `<!-- It's for you to complete on your own. -->` Check here for more to do.

**Comment by mel-d** [comment]
Ahh, I thought that was for a future assignment. I'll get on that now.

**Comment by mel-d** [comment]
I'm trying to add the all_adjusted_scored to the table on the welcome page. Something I did totally broke the code, and I can't find it for the life of me. I can't find anything wrong with the welcome page table or within the overall_grade_adjusted method.

**Comment by sumeetjain** [comment]
> Something I did totally broke the code

What's happening specifically?

**Comment by mel-d** [comment]
NoMethodError at /
undefined method `each' for 17.04:Float

```
file: student.rb location: overall_grade_adjusted line: 25 
```

**Comment by mel-d** [comment]
Although the method that this error message is referring to hasn't been changed at all

**Comment by sumeetjain** [comment]
```
NoMethodError at /
undefined method each' for 17.04:Float

    file: student.rb location: overall_grade_adjusted line: 25 
```

That **is** a good error. I think you should follow it through. The method you're citing is where Ruby discovered the error; but since methods call other methods, it's possible that another method (maybe `all_adjusted_scores`) is broken.

**Comment by mel-d** [comment]
Added the extra table info

**Comment by sumeetjain** [comment]
Yup, finished!

**Pull request closed without merging**